### DISCUSSION TITLE
Click to change space in Sonoma 14.1

### DISCUSSION THREAD

**Discussion by hamadahiro** [FelixKratz/SketchyBar#441] [Q&A] [answered]
I can't get clicking on the space icons in the bar to work, so I suspect it's a problem with my macOS version. Can someone confirm if this works on macOS Sonoma 14.1?

**Comment by FelixKratz** [comment, marked as answer]
Update to the latest yabai version for this to work again.

**Reply by hamadahiro** [reply]
Did it the other day. It works now!
